package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
- [ ] Kindly make sure to add `react-router-dome` to your project so you can make the routes in this project